src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -368,15 +397,14 @@
         currentActiveGroup?.classList.remove('ibexa-collapse--active-field-definitions-group');
         group.classList.add('ibexa-collapse--active-field-definitions-group');
     };
-    const removeLoadingField = () => {
```

Comment:
This function had same problem as removeHighlight, but as it was used only in one place, it was replaced with `field.remove()` instead in call fragment